app/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+services:
+  flaskapp:
+    build: 
+      context: ./ui
+      dockerfile: Dockerfile-ui
+    ports:
+      - "5090:5090"
+    networks:
+      - web_nw
+  nginx:
+    image: "nginx:1.13.5"
+    ports:
+      - "8080:80"
+    volumes:
+      - ./nginx:/etc/nginx/conf.d
+    networks:
+      - web_nw
+    depends_on: 
+      - flaskapp
+  elasticsearch:
```

Comment:
not suggesting to change anything now -  we need to, in future, also install plugins for elasticsearch in this config

Reply:
yeah good point. that would be done by creating our own dockerfile for ES that takes that base ES 5.x image and adds other stuff

can you please add Issues in this repo for the things that you'd like to see added?